lib/lotus/repository.rb

```diff
@@ -671,12 +672,30 @@ def _persisted?(entity)
         !!entity.id
       end
 
-      def _update_timestamps(entity)
+      # Add time create an entity
+      def _update_created_at(entity)
         if entity.class.attributes.include?(:created_at)
           entity.created_at ||= Time.now.utc
         end
       end
 
+      # Add time update an entity
+      def _update_updated_at(entity)
+        if entity.class.attributes.include?(:updated_at)
```

Comment:
@mymai91 @joneslee85 This implementation breaks one of the goals of `Lotus::Model`: persist simple objects.

```ruby
class Book
  attr_accessor :id
  def initialize(attributes = {})
  end
end

BookRepository.create(Book.new) # => NoMethodError: undefined method `attributes' for Book:Class
```

Can you please add a test case for this scenario and change the implementation to: if `entity.respond_to?(:updated_at=)`? Thank you.